src/main/java/github/scarsz/discordsrv/listeners/DiscordConsoleListener.java

```diff
@@ -82,7 +84,14 @@ public void onGuildMessageReceived(GuildMessageReceivedEvent event) {
 
         // log command to console log file, if this fails the command is not executed for safety reasons unless this is turned off
         try {
-            String fileName = DiscordSRV.config().getString("DiscordConsoleChannelUsageLog");
+            DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+            LocalDateTime now = LocalDateTime.now();
+            File dataFolder = DiscordSRV.getPlugin().getDataFolder();
+            if (!dataFolder.exists()) dataFolder.mkdir();
+            File logFolder = new File(dataFolder.getAbsolutePath() + File.separator + "logs");
+            if (!logFolder.exists()) logFolder.mkdir();
+            String fileName = logFolder + File.separator + "console-log-" + dtf.format(now) + ".log";
+
             if (StringUtils.isNotBlank(fileName)) {
```

Comment:
This check is very invalid now

Reply:
With the try/catch, later on, should I ignore the folder exists checks?

Reply:
what

Reply:
Sorry am not too familiar with Java, I don't understand what you mean by "This check is very invalid now".

Reply:
`fileName` is never blank, so that will always be `true`